boot-cw-plus/examples/cw-plus-mock.rs

```diff
@@ -0,0 +1,36 @@
+use boot_core::*;
+use boot_cw_plus::{Cw20QueryMsgFns, CwPlus};
+use cosmwasm_std::Addr;
+use cw20::Cw20Coin;
+
+// shows how to deploy CwPlus to a mock environment
```

Comment:
nit: these main functions should probably use `///` doc comment

Reply:
examples can't be imported so not something that's done 
i don't even think they are included when you run `cargo docs`